plugins/modules/user_role_workflow_manager.py

```diff
@@ -963,6 +963,33 @@ def validate_string_field(self, field_value, regex, error_message, error_message
         if field_value and not regex.match(field_value):
             error_messages.append(error_message)
 
+    def validate_password(self, password, error_messages):
+        """
+        Validate password from password and append error message if it does not match the criteria.
```

Comment:
I don't see the API arguments in docstring..
```
"""
   Validate the provided password and append an error message if it does not meet the criteria.

    Args:
        password (str): The password to be validated. Must be a string.
        error_messages (list): A list where error messages are appended if the password does not meet the criteria.

    Returns:
        None: This function does not return a value, but it may append an error message to `error_messages` if the password is invalid.

    Criteria:
        - The password must be 8 to 20 characters long.
        - The password must include characters from at least three of the following classes:
          lowercase letters, uppercase letters, digits, and special characters.
    """
```

Reply:
Updated the code accordingly